app/src/main/java/com/example/cinemaddict/component/GradientTextView.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.cinemaddict.component
+
+import android.content.Context
+import android.graphics.LinearGradient
+import android.graphics.Shader
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import com.example.cinemaddict.R
+
+class GradientTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : AppCompatTextView(context, attrs, defStyleAttr) {
+
+    private val gradientColors = intArrayOf(
```

Comment:
it would be better to be able use different gradient for textview

Reply:
fixed